### PR TITLE
Move Python examples

### PR DESCRIPTION
**Pull request by irvinebroque**

### CODE REVIEW

File: templates/python/fastapi/wrangler.toml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name = "hello-fastapi"
+main = "worker.py"
+compatibility_flags = ["python_workers"]
+compatibility_date = "2023-12-18"
```
Comment (irvinebroque) [inline templates/python/fastapi/wrangler.toml:4]:
```suggestion
compatibility_date = "2023-03-18"
```

File: templates/python/hello-world/wrangler.toml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+name = "hello-python-worker"
+main = "entry.py"
+compatibility_flags = ["python_workers"]
+compatibility_date = "2024-01-29"
```
Comment (irvinebroque) [inline templates/python/hello-world/wrangler.toml:4]:
```suggestion
compatibility_date = "2024-03-18"
```

File: templates/python/langchain-openai/wrangler.toml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+name = "langchain-python-worker"
+main = "worker.py"
+compatibility_flags = ["python_workers"]
+compatibility_date = "2024-01-29"
```
Comment (irvinebroque) [inline templates/python/langchain-openai/wrangler.toml:4]:
```suggestion
compatibility_date = "2024-03-18"
```

File: templates/python/langchain-openai/worker.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+from langchain_core.prompts import PromptTemplate
+from langchain_openai import OpenAI
+
+async def on_fetch(request, env):
+  prompt = PromptTemplate.from_template("Complete the following sentence: I am a {profession} and ")
+  llm = OpenAI(api_key=env.API_KEY)
+  chain = prompt | llm
+
+  res = await chain.ainvoke({"profession": "electrician"})
+  print(res)
```
Comment (dom96) [inline templates/python/langchain-openai/worker.py:11]:
```suggestion
  return Response.new(res.split(".")[0].strip())
```

File: templates/python/langchain-openai/requirements.txt [hunk 5]
```diff
@@ -0,0 +1,4 @@
+ssl
+langchain_core
+langchain_openai
+openaib
```
Comment (dom96) [inline templates/python/langchain-openai/requirements.txt:4]: Where did you get the `ssl`/`openaib` from? pretty sure only the two are needed

```suggestion
langchain_core
langchain_openai
```

File: templates/python/fastapi/requirements.txt [hunk 6]
```diff
@@ -0,0 +1,2 @@
+fastapi
+ssl
```
Comment (dom96) [inline templates/python/fastapi/requirements.txt:2]: is `ssl` needed here?

Comment (hoodmane) [reply templates/python/fastapi/requirements.txt:2]: No not anymore.

Comment (irvinebroque) [reply templates/python/fastapi/requirements.txt:2]:
```suggestion
```

**Comment by irvinebroque** [comment]
Closing in favor of `cloudflare/python-workers-examples`

**Pull request closed without merging**